tasks/install.yml

```diff
@@ -34,17 +34,28 @@
     cache_valid_time: "{{apt_cache_valid_time | default (3600)}}"
   with_items: ["python-psycopg2", "python-pycurl", "locales"]
 
-- name: PostgreSQL | Install PostgreSQL
+- name: PostgreSQL | Install PostgreSQL client
   apt:
     name: "{{item}}"
     state: present
     update_cache: yes
     cache_valid_time: "{{apt_cache_valid_time | default (3600)}}"
   environment: "{{postgresql_env}}"
   with_items:
-    - "postgresql-{{postgresql_version}}"
```

Comment:
I think it's possible rewrite with {{ 'postgresql-' + postgresql_version if postgresql_install_server else '' }}. For client you can write similar condition. Don't need task above.